startup-scripts.md

```diff
@@ -0,0 +1,308 @@
+## Overview
+
+KubeVirt supports the ability to assign a startup script to a Virtual Machine
+instance which is executed automatically when the Virtual Machine initializes.
+
+These scripts are commonly used to automate injection of users and ssh keys into
+Virtual Machines in order to provide remote access to the machine. For example,
+a startup script can be used to inject credentials into a Virtual Machine that
+allows an Ansible job running on a remote host to access and provision the
+Virtual Machine.
+
+Startup scripts are not limited to any specific use case though. They can be
+used to run any arbitrary script in a Virtual Machine on boot. 
+ 
+### Cloud-Init
+
+https://cloudinit.readthedocs.io/en/latest/
```

Comment:
Suggest using a markdown tag to make this a link. Also, this same link is below so might not be needed here.

Reply:
oops, I didn't copy the link twice. Good catch. i'll make it a markdown link too.